system_tests/seqcompensation_test.go

```diff
@@ -50,15 +50,17 @@ func TestSequencerCompensation(t *testing.T) {
 		Fail(t, "Unexpected balance:", l2balance)
 	}
 
-	initialSeqBalance, err := l2clientB.BalanceAt(ctx, l1pricing.SequencerAddress, big.NewInt(0))
+	initialSeqBalance, err := l2clientB.BalanceAt(ctx, l1pricing.BatchPosterAddress, big.NewInt(0))
 	Require(t, err)
 	if initialSeqBalance.Sign() != 0 {
 		Fail(t, "Unexpected initial sequencer balance:", initialSeqBalance)
 	}
-	finalSeqBalance, err := l2clientB.BalanceAt(ctx, l1pricing.SequencerAddress, nil)
+	finalSeqBalance, err := l2clientB.BalanceAt(ctx, l1pricing.L1PricerFundsPoolAddress, nil)
 	Require(t, err)
-	if finalSeqBalance.Sign() <= 0 {
-		Fail(t, "Unexpected final sequencer balance:", finalSeqBalance)
-	}
+	// Comment out this check, because new scheme doesn't reimburse sequencer immediately
+	// if finalSeqBalance.Sign() <= 0 {
+	//	Fail(t, "Unexpected final sequencer balance:", finalSeqBalance)
+	//}
+	_ = finalSeqBalance
```

Comment:
Should we just remove this test? It seems like it doesn't really check anything now that we don't immediately reimburse the sequencer.

Reply:
Removed unneeded test.